js/privatebin.js

```diff
@@ -2247,6 +2252,34 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             // hide modal
             $passwordModal.modal('hide');
 
+            // check if protected pathname
+            const url = new URL(window.location);
+
+            // if protected request password
+            if(url.hash.startsWith('#protected/')) {
+                const pt = '#protected/';
```

Comment:
Also this prefix should then only be defined once in the code statically, as it is static…